exercises/wordy/canonical-data.json

```diff
@@ -151,6 +151,22 @@
       },
       "expected": {"error": "syntax error"}
     },
+    {
+      "description": "reject unstarted problem",
+      "property": "answer",
+      "input": {
+        "question": "What is?"
+      },
+      "expected": {"error": "syntax error"}
+    },
+    {
+      "description": "reject empty string",
```

Comment:
Is this test perhaps unnecessary because of #902?

Reply:
Part of me wants to say "but then who will guard against people assuming that the input is always at least two words!" but I guess part of the question is what assumptions we're at liberty to make. Here's my deal. I'm going to leave that case for later. If there's a desire to add it, it'll be in 1.6.0

Reply:
Perhaps we need a case that asks a valid question but doesn't start with "What is".  "What is" shouldn't influence the outcome.
```
5 plus 6 is?
```
should work the same as:
```
What is 6 plus 5?
```
and
```
6 plus 2 plus 3?
```
For another PR perhaps......

Reply:
These proposals, along with the empty string, do bring up interesting questions of how far we want to take the parsing aspect in this problem.

The arithmetic portion is not *too* complicated, but does bring up interesting questions of how to translate each operation from its verbal form to its code form. So there could be some room to do more parsing.

I have not yet decided what I would suggest in this realm.